src/gas/fetchGasEstimatesViaEthFeeHistory/BlockFeeHistoryDatasetFetcher.ts

```diff
@@ -0,0 +1,39 @@
+import { BN } from 'ethereumjs-util';
```

Comment:
The purpose of this class is to hold different calls to `eth_feeHistory` where all of the resulting datasets end with the same block but where the requested number of blocks and/or percentiles may be different. The goal is to consolidate requests as much as possible so we're not making duplicate requests, so by keeping everything in one place, my hope is that it is easier to keep track of and monitor that possible consolidation.